### PR TITLE
moyo-header

### PR DESCRIPTION
**Pull request by barsuchellos**
- [DEMO LINK](https://barsuchellos.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://barsuchellos.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,50 @@
...
+          </li>
+          <li>
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li>
+            <a
+          href="#Laptops & Computers"
+          data-qa="hover"
+          class="nav__link"
+            >
```
Comment (OleksiiBlezniuk) [inline src/index.html:36]:
```suggestion
            <a
              href="#Laptops & Computers"
              data-qa="hover"
              class="nav__link"
            >
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,53 @@
...
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 5px;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:12]: Do not use tags for styling

File: src/style.css [hunk 3]
```diff
@@ -1 +1,53 @@
...
+
+.nav-list {
+  display: flex;
+  width: 85%;
```
Comment (OleksiiBlezniuk) [inline src/style.css:16]: Why this value?

File: src/style.css [hunk 4]
```diff
@@ -1 +1,53 @@
...
+  align-items: center;
+  padding: 0 50px 0 0;
+}
+
+.nav__link {
+  box-sizing: border-box;
+  font-family: "Roboto" , sans-serif;
```
Comment (OleksiiBlezniuk) [inline src/style.css:23]: Usually we set font styles to `html` tag

File: src/style.css [hunk 5]
```diff
@@ -1 +1,53 @@
...
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav-list a:hover {
+  color: #00acdc;
+  transition: 0.3s;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:33]: 1 - do not use tags for styles
2 - no need to increase selector specifity, link class is enough
3 - set transition to link class, now it's not working when you remove cursor from link

File: src/style.css [hunk 6]
```diff
@@ -1 +1,53 @@
...
+
+.is-active {
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
```
Comment (OleksiiBlezniuk) [inline src/style.css:37]: No, it's shouldn't be implemented by border (this underline should have border-radius)
Use pseudoelement for this purpose

File: src/style.css [hunk 7]
```diff
@@ -1 +1,53 @@
...
+  padding: 22px 0 18px;
+}
+
+.logo {
+  width: 30%;
```
Comment (OleksiiBlezniuk) [inline src/style.css:42]: Why that value?

File: src/style.css [hunk 8]
```diff
@@ -1 +1,53 @@
...
+  height: 40px;
+  padding: 0 0 0 50px;
```
Comment (OleksiiBlezniuk) [inline src/style.css:44]:
```suggestion
  padding-left: 50px;
```

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Something is wrong with your layout 🙂

Screenshot 2022-08-17 at 11 45 38
 
Also lets try to fix some tests, for example links should have bigger clickable zone (you can use paddings for them)

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**